server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -70,6 +77,45 @@ def _get_single_condition_criterion(
 
         return super()._get_single_condition_criterion(condition, prev_step_name)
 
+    # As weird as it seems, bigquery reverse the 'from' and the 'until'
+    # operators tha'ts why we're going to overide that by reverse those
```

Comment:
Can you provide an actual example of this ?

Reply:
looks like the method `_get_single_condition_criterion` was reversed, i fixed that here : https://github.com/ToucanToco/weaverbird/pull/1413/commits/2753e65af7ca160014324a84a6fafbe4d64f0107